README.md

```diff
@@ -0,0 +1,419 @@
+## Certification-Service
+
+### Pre-requisites
+0. Make a copy of setVars.sh.sample to list all the environment vars needed.
```

Comment:
Remove step 0 to 3 this is not required now.

Reply:
done